src/zkevm_llvm/main.rs

```diff
@@ -45,11 +45,13 @@ fn main_inner() -> anyhow::Result<()> {
         }
         Arguments::Build {
             debug,
+            mut targets,
             enable_tests,
             enable_coverage,
             extra_args,
             use_ccache,
         } => {
+            targets.insert(0, "EraVM".to_owned());
```

Comment:
Does this mean we always build targeting EraVM?  That makes sense commercially right now, but I think our long-term goal is to be more general, so I’d suggest that we start treating targets uniformly in this change.

Reply:
I'm ok with this when we release the EVM BE, but currently it would break *all* existing places where the tool is used. At least 1 default target we must keep.